pkg/bgp/speaker.go

```diff
@@ -0,0 +1,96 @@
+package bgp
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	bgpconfig "github.com/cilium/cilium/pkg/bgp/config"
+	bgpk8s "github.com/cilium/cilium/pkg/bgp/k8s"
+	bgplog "github.com/cilium/cilium/pkg/bgp/log"
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+	nodetypes "github.com/cilium/cilium/pkg/node/types"
+	"github.com/cilium/cilium/pkg/option"
+	"go.universe.tf/metallb/pkg/k8s/types"
+	metallbspr "go.universe.tf/metallb/pkg/speaker"
```

Comment:
Our preferred ordering (with newlines in between) for imports is:

```
stdlib

cilium pkgs

third party pkgs
```

Reply:
Hmm, pretty sure "gopls" is popping these into the order shown. Let me see if i can manually reorder and gopls not move them around.

Reply:
Cool, gopls allows for this:

```
	"context"
	"fmt"
	"os"

	bgpconfig "github.com/cilium/cilium/pkg/bgp/config"
	bgpk8s "github.com/cilium/cilium/pkg/bgp/k8s"
	bgplog "github.com/cilium/cilium/pkg/bgp/log"
	"github.com/cilium/cilium/pkg/logging"
	"github.com/cilium/cilium/pkg/logging/logfields"
	nodetypes "github.com/cilium/cilium/pkg/node/types"
	"github.com/cilium/cilium/pkg/option"

	"go.universe.tf/metallb/pkg/k8s/types"
	metallbspr "go.universe.tf/metallb/pkg/speaker"
```

look cool?